bin/nodeschool-admin

```diff
@@ -27,13 +28,25 @@ var input = nopt({
 input.path = input.path || process.cwd()
 log.level = input.loglevel || 'info'
 
-if (!input['gh-client-id'] && process.env.GITHUB_CLIENT_SECRET && process.env.GITHUB_CLIENT_ID) {
-  input['gh-client-id'] = process.env.GITHUB_CLIENT_ID
-  input['gh-client-secret'] = process.env.GITHUB_CLIENT_SECRET
-}
-
-if (!input['gh-token'] && process.env.GITHUB_TOKEN) {
-  input['gh-token'] = process.env.GITHUB_TOKEN
+// Checking it Github variables exist in '/config/github.js'
+if (fs.exists('../config/github')) {
```

Comment:
This path is a rather random path. How about putting it in `~/.nodeschool-admin.json` ?